src/index.js

```diff
@@ -56,14 +56,16 @@ async function Main() {
         SmashLeague.getUnrankedPlayerScore(newRankingObj.ranking.length)
     )
 
+    const newChampion = Utils.aNewChampion(Ranking.ranking, newRankingObj.ranking)
 
     const blocksToPost = SmashLeagueInteractions.getUpdatesToNotifyUsers(
         isItTimeToCommit ? {/* TODO: send weeks times */} : null,
         activities.reportedResults.length,
         ignoredActivities, 
         activities.ignoredMessages.length,
         newRankingObj.season,
-        IS_UPDATE ? version : null
+        IS_UPDATE ? version : null,
+        newChampion
```

Comment:
This function is starting to grow... We use the first argument to pass data that is related to the end of the week. So please send the _newChampion_ data as part of the first argument. *Check suggestion*